test/libsolidity/semanticTests/call_forward_bytes.sol

```diff
@@ -0,0 +1,26 @@
+contract receiver {
+    uint256 public received;
+    function recv(uint256 x) public { received += x + 1; }
+    fallback() external { received = 0x80; }
+}
+contract sender {
+    constructor() { rec = new receiver(); }
+    fallback() external { savedData = msg.data; }
+    function forward() public returns (bool) { address(rec).call(savedData); return true; }
+    function clear() public returns (bool) { delete savedData; return true; }
+    function val() public returns (uint) { return rec.received(); }
+    receiver rec;
+    bytes savedData;
+}
+// ====
+// compileViaYul: also
+// ----
+// () ->
+// recv(): 7 ->
+// val() -> 0
+// forward() -> true
+// val() -> 0x80
+// clear() -> true
+// val() -> 0x80
+// forward() -> true
+// val() -> 0x80
```

Comment:
Would be best to configure your editor to always keep a newline at the end. Otherwise it annoyingly changes back and forth in diffs when people edit the same file.

Reply:
Thanks for pointing out! Will take care of the same in the future